selfdrive/loggerd/loggerd.cc

```diff
@@ -276,13 +356,18 @@ void encoder_thread(bool is_streaming, bool raw_clips, int cam_idx) {
           s.finish_close += 1;
           pthread_mutex_unlock(&s.rotate_lock);
 
-          while(s.finish_close > 0 && s.finish_close < s.num_encoder) {
-            // printf("%d waiting for others to actually close, %d/%d \n", my_idx, s.finish_close, s.num_encoder);
-            s.cv.wait(lk);
-          }
+          while(s.finish_close > 0 && s.finish_close < s.num_encoder && !do_exit) { usleep(1000); }
```

Comment:
It seems that incoming camera frames may be lost while waiting for other encoders to complete. will this be a problem?

Reply:
sorry,forget about it.I got it wrong, it's yuv stream from pool, not rgb from tbuffer

Reply:
modeld may also encounter the same problem. https://github.com/commaai/openpilot/issues/2146